microsoft-edge/extensions-chromium/publish/api/addons-api-reference.md

```diff
@@ -42,21 +54,25 @@ Uploads a package to update an existing draft submission of an add-on product.
 |---|---|
 | `productID` | Required.  The product ID of the product to which the package must be uploaded. |
```

Comment:
```suggestion
| `$productID` | Required.  The product ID of the product to which the package must be uploaded. |
```
in these tables, change eg productID to $productID ?
@yvsubhash 

consistency pass - live docs use 3-4 different notations for seemingly same variable

Reply:
Ans: We can standardize "productID" and "operationID" var notation to same format consistently throughout using .md & ref .md